src/Resources/views/storefront/page/search/search-pagelet.html.twig

```diff
@@ -27,7 +32,7 @@
                 dataUrl: dataUrl,
                 filterUrl: '',
                 params: { search: page.searchTerm },
-                sidebar: 0,
+                sidebar: isInSidebar ? 1 : 0,
```

Comment:
What if you default `isInSidebar` to `false`. Do you still need that?

Reply:
Yes apparently, as it needs an integer for some reason.